index.md

```diff
@@ -0,0 +1,29 @@
+<!-- Note: 'N' will be grayed out in all cases -->
+┏━┓┏━┓┏┓╻┏━┓╻ ╻┏━┓╻ ╻╺┓ 
+┣━┫┗━┓┃┗┫┣━┓┗━┫┏━┛┗━┫ ┃ 
+╹ ╹┗━┛╹ ╹┗━┛  ╹┗━╸  ╹╺┻╸
+--- Outside the door ---
+
+You read the large neon sign hanging over the door of the Seattle-based 
+[peering establishment][sigil-peering] you've come across.
+
+The neon 'N's light flickers wearily and gives up.
+
+You knock, and a young lad opens the door and flashes a grin.
+
+"Wobscale Technologies, I'm Jason, how may I take your order?"
+
+[Choose][about-1] -- What can you tell me about this AS64241 operation?
```

Comment:
There should probably be a "Go West" option which puts you on rails to being eaten by a grue.